app/api/routes.py

```diff
@@ -25,6 +28,27 @@ def get_plant(id):
     response.status_code = 200
     return response
 
+@api.route('/garden', methods=['POST'])
+def add_to_garden():
+    plant_id = request.args['plant_id']
+    plant_name = request.args['plant_name']
+
+    plant = Plant.query.filter_by(id=plant_id,).first()
+    days_to_harvest = (datetime.now() + timedelta(days=plant.harvest_time))
+
+    garden = Garden(plant_id=plant_id,plant_name=plant_name)
+    db.session.add_all([garden])
+    db.session.commit()
```

Comment:
I miss Rails naming... and Database interactions!  Never thought I'd be longing for ActiveRecord!